content/about/about.html

```diff
@@ -15,48 +15,48 @@
   <body>
     <main>
       <h1>About</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/7f54d9620e32231021aad607b7af7eeee5c24f54 -->
       <ul>
+        <li lang="ja">
```

Comment:
`html`に`lang="ja"`指定済みなので、ここには不要ですね
```suggestion
        <li>
```